### ISSUE TITLE
missing Relay

### ISSUE THREAD

**Issue by sibelius** [GantMan/ReactStateMuseum#8]
https://github.com/facebook/relay

Relay handles state and data fetching

we could also manage local state using client schema extensions

**Comment by GantMan** [comment]
Yes!   @sibelius - would you be interested in adding that in a PR?
